### PR TITLE
An option to ingest ee asset as a table/featureCollection

### PR DESCRIPTION
**Pull request by deepgabani8**

### CODE REVIEW

File: weather_mv/README.md
Comment (alxmrs) [inline weather_mv/README.md:326]: Please indicate the default value.

File: weather_mv/loader_pipeline/ee.py
Comment (alxmrs) [inline weather_mv/loader_pipeline/ee.py:234]: Let's use choices instead of a string. 

https://docs.python.org/3/library/argparse.html#choices

**Review by alxmrs — COMMENTED** [review]
Do feature collections consume COGs? Does this work e2e (can I see the output)?

**Comment by deepgabani8** [comment]
@alxmrs 
I missed that point. Feature collection wouldn't be able to consume COGs.
We shouldn't merge this here then, right?

**Comment by alxmrs** [comment]
No, let's not merge this. I'd like to see the results of e2e prototypes before we land this change.

File: weather_mv/loader_pipeline/ee.py
Comment (alxmrs) [inline weather_mv/loader_pipeline/ee.py:527]: Optional style nit: My personal preference is that extracting out these variables is unnecessary; you can just access them from `asset_data`.

**Review by alxmrs — APPROVED** [review]
LGTM. One ask: Please file a bug s.t. we can scale EE ingestion to large files (which are bottlenecked on memory).

File: weather_mv/loader_pipeline/ee.py
Comment (deepgabani8) [reply weather_mv/loader_pipeline/ee.py:443]: Yes. Added [this bug](https://github.com/google/weather-tools/issues/228).

**Pull request merged**
